src/jenkins_job_insight/main.py

```
@@ -1192,6 +1259,261 @@ async def enrich_comments(
     return {"enrichments": enrichments}
 
 
+@app.post("/results/{job_id}/preview-github-issue")
+async def preview_github_issue(
+    job_id: str,
+    body: PreviewIssueRequest,
+    request: Request,
+    settings: Settings = Depends(get_settings),
```

Ruff (0.15.6)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)